platform/linux-generic/odp_fdserver.c

```diff
@@ -265,7 +265,9 @@ static int get_socket(void)
 	remote.sun_family = AF_UNIX;
 	strcpy(remote.sun_path, sockpath);
 	len = strlen(remote.sun_path) + sizeof(remote.sun_family);
-	if (connect(s_sock, (struct sockaddr *)&remote, len) == -1) {
+	while (connect(s_sock, (struct sockaddr *)&remote, len) == -1) {
+		if (errno == EINTR)
+			continue;
 		ODP_ERR("cannot connect to server: %s\n", strerror(errno));
```

Comment:
I was thinking if we should account for other errors, like `ETIMEDOUT` or `ECONNREFUSED` (the server's backlog is set to 2 and could happen in busy situations).
These are unlikely events, of course, but they may need handling nonetheless.